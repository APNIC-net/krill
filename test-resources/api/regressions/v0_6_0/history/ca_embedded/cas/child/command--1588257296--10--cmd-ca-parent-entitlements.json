{
  "actor": "krill",
  "time": "2020-04-30T14:34:56.756963Z",
  "handle": "child",
  "version": 13,
  "sequence": 10,
  "details": {
    "update_resource_classes": [
      "ta",
      {}
    ]
  },
  "effect": {
    "events": [
      13
    ]
  }
}