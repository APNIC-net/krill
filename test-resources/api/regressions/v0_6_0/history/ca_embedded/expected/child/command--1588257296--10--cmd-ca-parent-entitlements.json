{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:34:56.756963Z",
    "handle": "child",
    "version": 13,
    "sequence": 10,
    "details": {
      "update_resource_classes": [
        "ta",
        {}
      ]
    },
    "effect": {
      "events": [
        13
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "child",
        "version": 13,
        "details": {
          "resource_class_removed": [
            "0",
            {
              "ca_repo": "rsync://localhost/repo/child/0",
              "added": [],
              "updated": [],
              "withdrawn": [
                {
                  "name": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85.mft",
                  "hash": "b7fcd682a12a7b5c11dc0c58d5630665904c8b5fcbabd295b2f53ffb6898759d"
                },
                {
                  "name": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85.crl",
                  "hash": "73884498a6ec3a01febee9814085f5d0dc082cd5e99f29f5f214b4a4d8ef0c7a"
                }
              ]
            },
            "ta",
            [
              {
                "class_name": "0",
                "key": "C4CF00E42E046EBE5275589D3462F1B1F43FBA85"
              }
            ]
          ]
        }
      }
    ]
  }
}